### PR TITLE
update client page

### PR DESCRIPTION
**Pull request by jospoortvliet**
Ideally some thoughts and review by @rullzer @marinofaggiana @tobiasKaminsky @mario and, well, everyone who cares about the client ;-)

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

clients_small

### CODE REVIEW

**Comment by tflidd** [comment]
For the key features, I would use bullet points for each feature in the list.

**Comment by tflidd** [comment]
Small typo in the ios-client section:
https://github.com/nextcloud/nextcloud.com/pull/703/files#diff-01b10886b01d6ce2cf5bb2ff58bfc7b2R91
> The client us kept always up to date with the latest iOS versions

The client is kept ...

File: page-clients.php [hunk 1]
```diff
@@ -1,49 +1,193 @@
...
+                <li>Share files with others</li>
+                <li>Shared files view</li>
+                <li>Simple contacts backup & restore</li>
+                <li>Auto-configuration of Contacts and Calendar integration through DAVDroid</li>
+                <li>Client-side, <a class="hyperlink" href="/endtoend">End-to-end encryption</a> (version >2.1)</li>
+                <li>Local and server-side search</li>
+                <li>Support for server push notifications and file activity feed</li>
+                <li>Multi-account support</li>
+                <li>Fingerprint locking</li>
+                <li>Adjusts theming to your Nextcloud server</li>
```
Comment (MariusBluem) [inline page-clients.php]: *based on your Nextcloud-servers theming preferences

File: page-clients.php [hunk 2]
```diff
@@ -1,49 +1,193 @@
...
+                <li>Edit files on the phone built in or using other apps installed on your iPhone or iPad.</li>
+                <li>Add, rename, copy, move and delete files and folders</li>
+                <li>Keep your favorite files synchronized offline</li>
+                <li>Share files with others</li>
+                <li>Client-side, <a class="hyperlink" href="/endtoend">End-to-end encryption</a> (version >3.0)</li>
+                <li>Local and server-side search</li>
+                <li>Support for server notifications and file activity feed</li>
+                <li>3D Touch peek&pop and homescreen quick actions</li>
+                <li>Multi-account support</li>
+                <li>Adjusts theming to your Nextcloud server</li>
```
Comment (MariusBluem) [inline page-clients.php]: *based on your Nextcloud-servers theming preferences

File: page-clients.php [hunk 3]
```diff
@@ -1,49 +1,193 @@
...
+                <li>Auto-upload of photos and videos with:
+                <ul>
+                <li>optional automatic folder categorization based on date taken</li>
+                <li>Optional wifi-only upload</li>
+                <li>Custom file naming</li>
+                <li>Choose JPG/HEIC</li>
```
Comment (MariusBluem) [inline page-clients.php]: *chose between JPG and HEIC for photo uploads

File: page-clients.php [hunk 4]
```diff
@@ -1,49 +1,193 @@
...
+            </div>
+            <div class="col-md-8 revealOnScroll feature--block">
+                <p class="section--paragraph"><?php echo $l->t('The iOS client has the following key features:
+                <ul>
+                <li>Display documents, photos, videos, audio files, as a list or thumbnails, sorted as you like.</li>
+                <li>Edit files on the phone built in or using other apps installed on your iPhone or iPad.</li>
+                <li>Add, rename, copy, move and delete files and folders</li>
+                <li>Keep your favorite files synchronized offline</li>
+                <li>Share files with others</li>
+                <li>Client-side, <a class="hyperlink" href="/endtoend">End-to-end encryption</a> (version >3.0)</li>
```
Comment (MariusBluem) [inline page-clients.php]: I would remove the „client-side“ here since this is included in the term of E2E

File: page-clients.php [hunk 5]
```diff
@@ -1,49 +1,193 @@
...
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-6 revealOnScroll image--floated">
+                <div data-type="youtube" data-video-id="5yFKZze6TqM"></div>
+            </div>
+            <div class="col-md-6 revealOnScroll feature--block">
+                <p class="section--paragraph__tittle"><?php echo $l->t('top-rated iOS app');?></p>
+                <p class="section--paragraph"><?php echo $l->t('The Nextcloud iOS app provides a way for users to access and sync files on the go in a easy and intuitive interface. The client us kept always up to date with the latest iOS versions and designed for both smaller and newer iPhones, iPhone X and iPad.');?></p>
+                <div class="row">
```
Comment (MariusBluem) [inline page-clients.php]: The iPhone X is no separate product line and is included in the „newest“ iPhone-term.

**Comment by tobiasKaminsky** [comment]
👍 apart from the suggested changes above I am fin :-)

**Comment by marinofaggiana** [comment]
E2E available from iOS V 2.20.0 ... but I want remake the image and Video ...

File: page-clients.php (hunk 4 above)
Comment (jospoortvliet) [reply page-clients.php]: I use client-side consistently to differentiate us from the fake security offered by a certain other solution which decrypts in the browser -> that's not a 'client'. After all, end-to-end is kind'a meaningless as it depends on what 'end' you define.

File: page-clients.php (hunk 3 above)
Comment (jospoortvliet) [reply page-clients.php]: done

File: page-clients.php (hunk 2 above)
Comment (jospoortvliet) [reply page-clients.php]: too long. I made it "Adjusts to Nextcloud server theming preferences"

File: page-clients.php (hunk 1 above)
Comment (jospoortvliet) [reply page-clients.php]: made it "Adjusts to Nextcloud server theming preferences" to keep it from getting crazy long

File: page-clients.php (hunk 5 above)
Comment (jospoortvliet) [reply page-clients.php]: But it is hot so I mention it separate for now ;-)

**Comment by jospoortvliet** [comment]
@marinofaggiana send them to me when you have them ;-) note that I can edit them so don't spend too much time on them!

**Comment by jospoortvliet** [comment]
@tflidd fixed the typo. Wrt bullets, I prefer them too but our designer @Espina2 has removed them and I won't be the guy putting them back 😸

**Comment by jospoortvliet** [comment]
Looks like nobody disagreed so - moving forward ;-)

Pls, if you're OK with a PR, do add an approval-review so I don't have to force-merge.

**Comment by rullzer** [comment]
Well like I told you by mail. The version number for the desktop client is not fixed at all. So maybe just put TBD there or something?

**Pull request merged**